### PR TITLE
Add symbolic icons support and ability to remove button labels

### PR DESCRIPTION
**Pull request by Suzie97**

### CODE REVIEW

File: src/Services/ActionManager.vala [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+ */
+
+public class Photostat.Services.ActionManager : Object {
+    public weak Photostat.Application app { get; construct; }
+    public weak Photostat.Window window { get; construct; }
+
+    public SimpleActionGroup actions { get; construct; }
+
+    public const string ACTION_PREFIX = "win.";
+    public const string ACTION_PREFERENCES  = "action_preferences";
```
Comment (Abdallah-Moh) [inline src/Services/ActionManager.vala:30]: Lint: fix  double spaces before `=`

**Review by Abdallah-Moh — CHANGES_REQUESTED** [review]

File: src/Services/ActionManager.vala (hunk 1 above)
Comment (Abdallah-Moh) [reply src/Services/ActionManager.vala:30]: at line 30

File: data/gschema.xml [hunk 2]
```diff
@@ -36,5 +36,11 @@
...
     </key>
+
+    <key name="show-label" type="b">
+      <default>true</default>
+      <summary>Show button labels</summary>
+      <description>Show or Hide the button labels to slim the UI.</description>
+    </key>
   </schema>
-</schemalist>
+</schemalist>
```
Comment (Abdallah-Moh) [inline data/gschema.xml:46]: Remove indentation at the end of line

File: src/Dialogs/SettingsDialog.vala [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+        }
+    }
+
+    private class SettingsSwitch : Gtk.Switch {
+        public SettingsSwitch (string setting) {
+            halign = Gtk.Align.START;
+            settings.bind (setting, this, "active", SettingsBindFlags.DEFAULT);
+        }
+    }
+
```
Comment (Abdallah-Moh) [inline src/Dialogs/SettingsDialog.vala:103]: Is there any way to hide `on` and `off` labels in switch

**Comment by Abdallah-Moh** [comment]
Menu label not hiding, and using symbolic icons shows a line under button
Screenshot from 2021-06-22 15 10 56

**Comment by Abdallah-Moh** [comment]
Align things to the center

**Comment by Suzie97** [comment]
> Menu label not hiding, and using symbolic icons shows a line under button
> Screenshot from 2021-06-22 15 10 56

That button is not yet fully implemented. That is why, it doesn't behave normally.

File: src/Dialogs/SettingsDialog.vala (hunk 3 above)
Comment (Suzie97) [reply src/Dialogs/SettingsDialog.vala:103]: I'm sorry, what label are you talking about?

**Comment by Abdallah-Moh** [comment]
The line also shows under the preference button

File: src/Dialogs/SettingsDialog.vala (hunk 3 above)
Comment (Abdallah-Moh) [reply src/Dialogs/SettingsDialog.vala:103]: The switch has labels inside it

Comment (Abdallah-Moh) [reply src/Dialogs/SettingsDialog.vala:103]: image

Comment (Suzie97) [reply src/Dialogs/SettingsDialog.vala:103]: No, not in Odin. Maybe a problem with eOS 5

**Pull request merged**